src/lib/baseCommand.ts

```diff
@@ -93,6 +93,14 @@ export default class Command {
     Command.debug(`opts: ${JSON.stringify(opts)}`);
 
     if (this.args.some(arg => arg.name === 'key')) {
+      if (opts.key && configstore.get('apiKey') === opts.key) {
+        info(
+          `You're currently making updates to the project: ${configstore.get(
+            'project'
+          )} on the account ${configstore.get('email')}`
+        );
```

Comment:
Copy edit[^1] below! Also note I added the `false` since we want to use this key emoji prefix (🔑) rather than ℹ️

```suggestion
        info(
          `🔑 ${configstore.get('email')} is currently logged in, using the stored API key for this project: chalk.blue(${configstore.get(
            'project'
          )})`, false
        );
```

[^1]: you'll definitely have to rerun prettier here, assuming the syntax is even correct lol